packages/core/src/compiler.ts

```diff
@@ -303,26 +320,50 @@ export default class Compiler {
     }
   }
 
-  setFormatter(key: string) {
-    if (this.runtimeIncludes(key, 'formatter')) return;
-    let cf = this.options.customFormatters[key];
+  setFormatter(runtimeFormatterKey: string, originalFormatterKey?: string) {
```

Comment:
e.g. `setFormatter("phone__esUS", "phone")`

Reply:
These arguments would be clearer for me with reversed order, i.e. "the `key` of the thing we're importing", and optionally "the `alias` we use for it internally". I'm also not a fan of really long names for local variables, esp. when combined with our 80 char width style.